### PR TITLE
Add release note for 0.243.2

### PR DESCRIPTION
**Pull request by rongrong**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.238.rst [hunk 1]
```diff
@@ -7,6 +7,9 @@ Release 0.238
    There is a bug in this release that will cause certain queries with lambda expressions to fail.
    There is a reliability issue in ZSTD compression that causes frequent excessive GC events in this release.
 
+.. warning::
+    There is a bug in LambdaDefinitionExpression canonicalization introduced in this release. For more details, go to {:issue:`15424`}.
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/release/release-0.238.rst:11]: I don't think we need the surrounding`{}`. Remove? Just
```
For more details, go to :issue:`15424`.
```
same here and below.

File: presto-docs/src/main/sphinx/release/release-0.243.2.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.243.1
+===============
+
+General Changes
+------------
+General Changes
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/release/release-0.243.2.rst:7]: Copy-paste error? Remove the duplicated title. Also, the number hyphens need to match the number of the characters on the last line.

File: presto-docs/src/main/sphinx/release/release-0.243.2.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.243.1
+===============
+
+General Changes
+------------
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/release/release-0.243.2.rst:6]: One of the comments is not addressed yet. You'll need 3 more hyphens here, or else it's a Sphinx syntax error.

**Review by caithagoras — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-0.243.2.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.243.1
```
Comment (caithagoras) [inline presto-docs/src/main/sphinx/release/release-0.243.2.rst:2]: Should be `Release 0.243.2`

Comment (rongrong) [reply presto-docs/src/main/sphinx/release/release-0.243.2.rst:2]: 🤦‍♀️

**Pull request merged**